common/templates/student/edx_ace/accountactivation/email/body.html

```diff
@@ -27,13 +27,12 @@
             padding: 20px;
             ">
 
-            <img src="https://mylearningx.pearson.com/static/pearson-pols-theme/images/PearsonLogo_Primary_Blk_RGB.png" style="width: 150px; padding: 5px; margin: auto;" />
+            <img src="https://partner-courses.pearson.com/static/pearson-theme/images/logo.png" style="width: 150px; padding: 5px; margin: auto;" />
 
             <div style="width: 90%;
-                        background-color: #D4EAE4;
                         text-align: left;
                         padding: 2% 5%;
-                        background-image: url('https://mylearningx.pearson.com/static/pearson-pols-theme/images/laptop_transparent-bg.png');
+                        background-color: #D4EAE4;
```

Comment:
Why is now a color instead of an image?. @Serafin-dev

Reply:
I removed the background-image. The background-color already existed, I just moved the line down to align it with the other background styles.